README.md

```diff
@@ -1,23 +1,39 @@
 # CS Club Website
 
-## Development
+---
 
-1. Install dependencies
+This is the official repository for the University of Adelaide Computer Science Club (CS Club) website. It is built using [Next.js](https://nextjs.org/), [React](https://reactjs.org/), [TypeScript](https://www.typescriptlang.org/), and [Tailwind CSS](https://tailwindcss.com/).
+
+## Getting Started
+
+To get started, please follow these steps: 
+1. Clone the repository and open the folder.
 
 ```bash
-pnpm i
+git clone https://github.com/compsci-adl/website
+cd website
 ```
 
-2. Rename `.env.local.example` to `.env.local` and edit it
+2. Install the dependencies.
+```bash
+pnpm i
+```
 
-3. Initialize database
+3. Set up the keys by copying `.env.local.example` to a new file `.env.local` and replace the placeholder keys with the actual keys. Please contact the open source officers on the CS Club Discord to get these keys.
 
+4. Initialise the database
 ```bash
-pnpm run db:push
+pnpm db:push
```

Comment:
Should be `pnpm run db:push`

Reply:
Same as before, `pnpm db:push` works the same as `pnpm run db:push`.